lib/vdsm/storage/exception.py

```diff
@@ -115,6 +115,22 @@ def __init__(self, volume, *args):
             self.value += list(args)
 
 
+class VolumeLVMCommandError(StorageException):
```

Comment:
Not sure that Volume is relevant - this should be used for vgs, pvs, and lvs.
I think the common property for this base class is holding a LVMCommandError
and formatting the error using key=value format.

So maybe call it _HoldingLVMCommandError?

This makes it clear that this is not meant to be created or raise, but only
used a a base class.

Reply:
I like the name, but the underscore at the beginning may be misinterpreted, I think. I would generally consider a class starting with an underscore to be internal (not to be imported), but not abstract.

Maybe we can let the code be more clear with it and overwrite the `__new__` method to check for the `cls` type, as in:
```python
    def __new__(cls, *args, **kwargs):
        # Base class cannot be directly instantiated.
        if cls is HoldingLVMCommandError:
            raise TypeError(
                f"{cls.__name__} exception is not meant to be directly instantiated.")
        return object.__new__(cls, *args, **kwargs)
```

Reply:
I don't see the need for this complication. Using _Prefix makes it clear that you should not use this class outside of the module. We can assume that oVirt developers do not use private stuff from other modules.